apps/web/src/pages/ApplicantDashboardPage/components/ReviewApplicationPreviewList.tsx

```diff
@@ -0,0 +1,576 @@
+        name {
+          en
+          fr
+          localized
+        }
+      }
+      language {
+        value
+        label {
+          en
+          fr
+          localized
+        }
+      }
+      department {
+        name {
+          en
+          fr
+          localized
+        }
+      }
+      closingDate
+      opportunityLength {
+        value
+        label {
+          en
+          fr
+          localized
+        }
+      }
+      isRemote
+      location {
+        en
+        fr
+        localized
+      }
+      securityClearance {
+        value
+        label {
+          en
+          fr
+          localized
+        }
+      }
+      areaOfSelection {
+        value
+        label {
+          en
+          fr
+        }
+      }
+      processNumber
+      poolSkills {
+        id
+        type {
+          value
+          label {
+            en
+            fr
+          }
+        }
+        skill {
+          id
+          key
+          name {
+            en
+            fr
+            localized
+          }
+          category {
+            value
+            label {
+              en
+              fr
+            }
+          }
+        }
+      }
+      screeningQuestions {
+        id
+      }
+    }
+  }
+`);
+
+interface ReviewApplicationDialogProps {
+  applicationQuery: ReviewApplicationPreviewListFragment;
+}
+
+const ReviewApplicationDialog = ({
+  applicationQuery: application,
+}: ReviewApplicationDialogProps) => {
+  const intl = useIntl();
+  const locale = getLocale(intl);
+  const paths = useRoutes();
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+
+  const pool = application?.pool;
+
+  const nullMessage = intl.formatMessage(commonMessages.notFound);
+  const poolName =
+    pool.name?.localized ?? intl.formatMessage(commonMessages.notFound);
+
+  // Separate essential and asset skills, sort them by category, and confirm they include skill data
+  const poolSkills = unpackMaybes(pool?.poolSkills);
+  const essentialPoolSkills = sortPoolSkillsBySkillCategory(
+    poolSkills.filter(
+      (poolSkill) => poolSkill.type?.value === PoolSkillType.Essential,
+    ),
+  );
+  const nonessentialPoolSkills = sortPoolSkillsBySkillCategory(
+    poolSkills.filter(
+      (poolSkill) => poolSkill.type?.value === PoolSkillType.Nonessential,
+    ),
+  );
+
+  const focusOnRecruitment = useRef(false);
+
+  const status = getApplicationStatusChip(
+    application.submittedAt,
+    pool.closingDate,
+    application.removedAt,
+    application.finalDecisionAt,
+    application.finalDecision?.value,
+    pool.areaOfSelection?.value,
+    application.assessmentStatus,
+    pool.screeningQuestions,
+    intl,
+  );
+
+  const isDraftStatus = status.value === ApplicationStatus.DRAFT;
+  const isExpiredStatus = status.value === ApplicationStatus.EXPIRED;
+  const lessThanThreeDaysTillClosingDate = pool?.closingDate
+    ? differenceInDays(Date.now(), parseDateTimeUtc(pool.closingDate)) < 3
```

Comment:
This doesn't seem to be calculating correctly - I think you need to reverse the parameters to differenceInDays.
![image](https://github.com/user-attachments/assets/a781baa5-eed6-4a05-8f91-d88a3187115d)

Reply:
Haha ya I saw that too, let me know if this is good [2797ae0](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12366/commits/2797ae0ec0795e3a4381dbf51e202d15142d31f7)

Reply:
I'm not sure this worked - in the list deadlines never appear in red even when within 3 days
![image](https://github.com/user-attachments/assets/b4014511-51f8-42d3-b43b-62ed234a7647)
and in the dialog for expired applications it shows "Received" instead of the deadline.
![image](https://github.com/user-attachments/assets/c026b49b-c611-4df9-8201-17422b3e5e34)

Reply:
@JoshBeveridge in the first image, is the "Deadline" date supposed to appear in red? I don't see it in the design.